articles/setting-up-cicd-for-python-packages-using-github-actions/index.md

```diff
@@ -0,0 +1,279 @@
+# Setting up CI/CD for Python Packages using GitHub Actions
+
```

Comment:
Before `Outline`, always add an introductory paragraph as shown in any of the articles [here](https://www.section.io/engineering-education/).

The introductory paragraph can contain:
- a brief of what this article is about
- what are the key takeaways from the article
- prerequisities for reading this article
- target audience